### PR TITLE
Refactoring: Add clearFilterWidget e2e helper

### PR DESCRIPTION
**Pull request by calherries**
This PR simply adds a new clearFilterWidget helper for e2e tests. 

It clears the filter widget value when there is one, by clicking this close icon:
image

### CODE REVIEW

File: e2e/support/helpers/e2e-ui-elements-helpers.js [hunk 1]
```diff
@@ -70,6 +70,10 @@ export function filterWidget() {
   return cy.get("fieldset");
 }
 
+export function clearFilterWidget(index = 0) {
+  return filterWidget().eq(index).find(".Icon-close").click();
```
Comment (WiNloSt) [inline e2e/support/helpers/e2e-ui-elements-helpers.js:74]: We use `.icon` to find our icons.

```suggestion
  return filterWidget().eq(index).icon("close").click();
```

File: e2e/test/scenarios/dashboard-filters/dashboard-filters-number.cy.spec.js [hunk 2]
```diff
@@ -44,7 +45,9 @@ describe("scenarios > dashboard > filters > number", () => {
           cy.findByText(representativeResult);
         });
 
-        clearFilter(index);
+        console.log(cy.state());
```
Comment (WiNloSt) [inline e2e/test/scenarios/dashboard-filters/dashboard-filters-number.cy.spec.js:48]: I suppose this should be deleted.

**Review by WiNloSt — APPROVED** [review]
left a few minor comments, otherwise, this looks great 👍

File: e2e/test/scenarios/dashboard-filters/dashboard-filters-number.cy.spec.js (hunk 2 above)
Comment (calherries) [reply e2e/test/scenarios/dashboard-filters/dashboard-filters-number.cy.spec.js:48]: I suppose you're right.

**Pull request merged**